core/src/repair_service.rs

```diff
@@ -385,6 +393,10 @@ impl RepairService {
                     })
                     .collect();
                 info!("repair_stats: {:?}", slot_to_count);
+                info!(
+                    "all shred repairs requested: {:?}",
```

Comment:
This represents all repair requests made, and will be of the format:

`{5: {1}}`

where this means we made a repair request for slot `5`, index `1`